lib/l10n-en_GB.js

```diff
@@ -22,6 +22,8 @@ exports.messages = {
 ,   "headers.h2-status.bad-h2": "Incorrect status h2 header."
     // headers/h2-toc
 ,   'headers.h2-toc.missing':  'There is no table of contents after the status section, labeled with a <code>&lt;h2&gt;</code> element with content <em>&ldquo;Table of Contents&rdquo;</em>.'
+    // headers/ol-toc
+,   'headers.ol-toc.not-found':  'The TOC should be an <code>&lt;ol class="toc"&gt;</code> inside the main navigation element (<code>&lt;nav id="toc"&gt;</code>).'
```

Comment:
For the keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED",  "MAY", I suggest we use `<span class="rfc2119">keyword</span>` like it appears in the old pubrules.